server/server.js

```diff
@@ -44,10 +44,10 @@ app.get("/api/v1/pet_type", (req, res)  =>  {
     })
 })
 
-app.get("/api/v1/:pet_type", (req, res)  =>  {
+app.get("/api/v1/:pet_type", (req, res) => {
   let petType = req.params.pet_type
-  if(petType == 'pigguinea' || petType == 'guineapig')  {
-    petType = 'guinea'
+  if (petType == "pigguinea" || petType == "guineapig") {
```

Comment:
I'm curious of the use case for getting "pigguinea" in your fetch. Can you clarify?

Reply:
readjusted to just take in petType.  were originally thinking if a user entered it incorrectly in the url when trying to get to the specific page